package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Js Project",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] It will be great to have all the necessary run commands in your project. This will improve your project configuration. For example: ⬇️ 
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
````
![scripts_requirement](https://user-images.githubusercontent.com/32361963/127466920-c6ae015e-76b3-4521-80b6-57fb91abe672.png)